Source/ZXing.Net.Mobile.Android/CameraExtensions.cs

```diff
@@ -0,0 +1,130 @@
+// <copyright company="APX Labs, Inc.">
+//     Copyright (c) APX Labs, Inc. All rights reserved.
+// </copyright>
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+// Many thanks to Jonathan Pryor from Xamarin for his assistance
+
+using System;
+using Android.Hardware;
+using Android.Runtime;
+
+namespace ApxLabs.FastAndroidCamera
```

Comment:
The reason I kept the APXLabs namespace in:

    CameraExtensions.cs
    FastJavaByteArray.cs
    FastJavaByteArrayEnumerator.cs

is that these files stems from: https://github.com/APXLabs/FastAndroidCamera. APXLabs is not my company, and even if the license permits me to change the namespace I felt it was a bit rude. 

I only modified the files to use the JniEnvironment class (introduced with Xamarin.Android 6.1, if I recall correctly)

However I see that James Athey (the original author - https://github.com/jamesathey) has done some simlar fixes this september, and even released a nuget: https://www.nuget.org/packages/FastAndroidCamera/ Perhaps it would be better to reference this, what do you think?